MapboxNavigation/MGLStyle.swift

```diff
@@ -1,24 +1,26 @@
 import Foundation
 import Mapbox
 
+
 extension MGLStyle {
     // Returns the URL to the current version of the Mapbox Navigation Guidance Day style.
     @objc
-    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: "mapbox://styles/mapbox/navigation-guidance-day-v2")! } }
+    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: (MGLAccountManager.hasChinaBaseURL ? MGLAccountManager.mapboxChinaDayStyleURL : "mapbox://styles/mapbox/navigation-guidance-day-v2"))! } }
```

Comment:
I agree with using constants for the China URLs since each gets reused. However, consider turning them into URL-typed static constants within MGLStyle. (There’s no need for string constants if we have constant URLs.)

```swift
extension MGLStyle {
    static let streetsChineseURL = URL(string: "mapbox://styles/mapbox/streets-zh-v1")!
}
```